scripts/reset-build-directories.sh

```diff
@@ -0,0 +1 @@
+rm -rf build/ .svelte-kit/ static/build/ static/fonts/
```

Comment:
The idea was to have the script called from various `npm` commands, so it seemed like a standalone script would suffice here. I can try doing this over in the Sveltekit config though; might work similarly to how Rollup works since the person who created Sveltekit also created Rollup!

Reply:
filed an issue to follow up on this one.